docs/storage/petalibrary/petalibrary.md

```diff
@@ -3,9 +3,15 @@
 ## General Access
 Each person who accesses the PetaLibrary is required to have a Research Computing account and Duo two-factor authentication. 
 
-PetaLibrary storage is presented as a file system directory under either:
+PetaLibrary storage is presented as a file system directory with the following path: 
+
+_For active & active+archive allocations:_
+```
+pl/active/<your_allocation_name>
+```
+
+_For archive & archive+DR allocations:_
```

Comment:
I think you misinterpreted my suggestion. I was not asking you to turn this into a table, I was asking you to make it tabs. I still think tabs would look better for these types of things. I also want everyone on the team to know how to do tabs. Would you mind turning this into tabs? See the section `Getting started with Open OnDemand` on the page `open_ondemand/index.md` to see the markdown syntax for doing this.

Reply:
`````
````{eval-rst}
.. tabs::

   .. code-tab:: Active & Active + Archive

        pl/active/<your_allocation_name>

   .. code-tab:: Archive & Archive + DR

        /pl/archive/<your_allocation_name>
````
`````

Reply:
Done!

Reply:
resolved